packages/generators/remove-generator.ts

```diff
@@ -0,0 +1,131 @@
+import * as fs from "fs";
+import * as path from "path";
+import Generator = require("yeoman-generator");
+
+import * as PROP_TYPES from "@webpack-cli/utils/prop-types";
+import { List } from "@webpack-cli/webpack-scaffold";
+
+/**
+ *
+ * Generator for removing properties
+ * @class	RemoveGenerator
+ * @extends	Generator
+ * @returns	{Void} After execution, transforms are triggered
+ *
+ */
+
+export default class RemoveGenerator extends Generator {
+	private configuration: {
+		config: {
+			configName?: string,
+			topScope?: string[],
+			webpackOptions?: any,
+		},
+	};
+	private webpackOptions: any;
+
+	constructor(args, opts) {
+		super(args, opts);
+		this.configuration = {
+			config: {
+				webpackOptions: {},
+			},
+		};
+
+		let configPath: string = path.resolve(process.cwd(), "webpack.config.js");
+		const webpackConfigExists: boolean = fs.existsSync(configPath);
+		if (!webpackConfigExists) {
+			configPath = null;
+			// end the generator stating webpack config not found or to specify the config
+		}
+		this.webpackOptions = require(configPath);
+	}
+
+	public getPropTypes(): string[] {
+		return Object.keys(this.webpackOptions);
+	}
+
+	public getModuleLoadersNames(): string[] {
+		if (this.webpackOptions.module && this.webpackOptions.module.rules) {
+			return this.webpackOptions.module.rules.map((rule: any) => rule ? rule.loader : null);
+		}
+	}
+
+	public prompting() {
+		const done: () => void | boolean = this.async();
```

Comment:
`(_)`

Reply:
If you do that then we'll have to pass something when we call `done()` at the end of generators.

Reply:
oh okay, let's keep it then (: